### PR TITLE
PM-10084: Move save button into navigation bar

### PR DESCRIPTION
**Pull request by matt-livefront**
## 🎟️ Tracking

[PM-10084](https://bitwarden.atlassian.net/browse/PM-10084)

## 📔 Objective

Moves the save or primary action button from the bottom of the scroll view into the navigation bar on the following screens:

- Add/edit vault item
- Add/edit send
- Add/edit folder
- Self-hosted environment setup
- Generator select username/password

## 📸 Screenshots

See snapshots.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-10084]: https://bitwarden.atlassian.net/browse/PM-10084?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by fedemkr — APPROVED** [review]
LGTM 🎉

File: BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/__Snapshots__/AddEditSendItemViewTests/test_snapshot_text_extension_withValues.1.png
Comment (KatherineInCode) [inline BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/__Snapshots__/AddEditSendItemViewTests/test_snapshot_text_extension_withValues.1.png:1]: 🤔 Is this situation (selecting a new account) one where we'd want to give the option for a save button?

Comment (matt-livefront) [reply BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/__Snapshots__/AddEditSendItemViewTests/test_snapshot_text_extension_withValues.1.png:1]: Good question! It looks like it works this way in the MAUI app and this is consistent with the vault list where you can still tap the add button if the profile switcher is visible.

Comment (KatherineInCode) [reply BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/__Snapshots__/AddEditSendItemViewTests/test_snapshot_text_extension_withValues.1.png:1]: If it's how it works in the MAUI app, I think we're good, then.

**Review by KatherineInCode — APPROVED** [review]

**Pull request merged**